membership-attribute-service/conf/riff-raff.yaml

```diff
@@ -7,12 +7,12 @@ deployments:
     parameters:
       templatePath: membership-attribute-service.yaml
       amiTags:
-        Recipe: xenial-membership
```

Comment:
there was an issue with the ami build on xenial, but there's no reason not to upgrade